src/main/webapp/i18n/en/student-dashboard.json

```diff
@@ -12,6 +12,8 @@
             "noStatistics": "No statistics available",
             "cardNoExerciseLabel": "No exercise planned",
             "cardExerciseLabel": "Next Exercise:",
+            "oldCourses": "Looking for old courses? Click",
+            "here": "here",
```

Comment:
can we move this in a more shared json. As for example in the ssh dialog we also use this word